.github/workflows/lint.yaml

```diff
@@ -12,3 +12,4 @@ jobs:
           chmod 700 get_helm.sh
           ./get_helm.sh
       - run: helm lint
+      - run: ./scripts/package.sh
```

Comment:
What should this do? This addition will just package the stuff but not actually commit the newly packaged stuff, right?

Reply:
Yeah it will not commit anything, just checking if the package.sh could be successfully run. This acts as a simple validation to see if there is any chart error.

Reply:
Ah, gotcha. We should come up with some way to validate the package.sh runs before something merges. That being said, doesn't have to be this PR.  I'll try to think something up.

Reply:
Yeah, we should verify that to make sure someone really run this. The problem with helm package is that this is non-deterministic as it always writes the timestamp each time it packages. So we will need to be smart about it in checking the diff.